issue-40-built-in-benchmarking-tool.md

```diff
@@ -0,0 +1,32 @@
+## Add Built-in Benchmarking Tool
+
+## Summary
+
+There is a need to support a built-in benchmarking tool in Submariner.
+This is to provide users with easy way to benchmark the network performance within and across clusters, and experiment with different tuning of the system.
+
+## Proposal
+
+Add a new ability to have a built-in benchmark tool.
+Implement a new Subctl command that will use iperf3 internally to benchmark the throughput between the two nodes.
+
+scheduling server and client can be placed within the same cluster or spread across multiple clusters.
+
+The user can specify source and destination nodes via node names. (Or via lables?)
```

Comment:
Personally, I prefer nodeNames here for simplicity, but the same can be achieved via selectors as well. 
Instead of taking "cluster2-worker" as input from user, we would take "kubernetes.io/hostname=cluster2-worker" (or any other label of choice). 

I'm fine with labels, but this needs to be properly documented as the same label can be applied on multiple nodes and nettest could be scheduled on one of them.